spec/reek/cli/application_spec.rb

```diff
@@ -0,0 +1,10 @@
+require_relative '../../spec_helper'
+require_lib 'reek/cli/application'
+
+RSpec.describe Reek::CLI::Application do
+  describe '#initialize' do
+    it 'works' do
+      Reek::CLI::Application.new []
+    end
```

Comment:
I had a reasoning (i.e., at least get it loaded so its coverage isn't zero), but I realize it's no better than having zero coverage.

I'll see if I can add some proper specs here and in the Task spec as well.